openapi/vonage.numberinsight/types.bal

```diff
@@ -240,17 +238,23 @@ public type NiResponseJsonAdvanced record {
     # If the user has changed carrier for `number`. The assumed status means that the information supplier has replied to the request but has not said explicitly that the number is ported.
     string? ported?;
     # Information about the roaming status for `number`. This is applicable to mobile numbers only.
-    NiRoaming|string? roaming?;
+    NiRoaming roaming?;
```

Comment:
@RolandHewage any reason to manually update  `roaming` to union of types `NiRoaming|string` in the initial commit ?
Not defined as `anyof` or `oneof` type in the OpenAPI file
```
        roaming:
          type: object
          description: "Information about the roaming status for number. This is applicable to mobile numbers only."
          x-nexmo-developer-collection-description-shown: true
          properties:
            status:
              type: string
              description: Roaming status for number 
              enum:
                - unknown
              example: unknown
              xml:
                attribute: true
```

Reply:
Since this PR was sent on early days that we started generating Open API connectors, I guess I was not aware about certain features we could use to modify the open API yaml. I apologize for that. But I agree and I'm aware now that this could be done by modifying the OpenAPI file with `anyof` or `oneof`.

Reply:
Updated the openapi file